Sources/DiscordKitCore/Objects/Data/Snowflake.swift

```diff
@@ -8,3 +8,17 @@
 import Foundation
 
 public typealias Snowflake = String
+
+public extension Snowflake {
+    func creationTime() -> Date {
```

Comment:
Complementing the suggestion below, this makes it the responsibility of the end-developer to handle decoding failures.

```suggestion
    func creationTime() -> Date? {
```